projects/igniteui-angular/src/lib/directives/date-time-editor/date-time-editor.directive.ts

```diff
@@ -450,6 +456,11 @@ export class IgxDateTimeEditorDirective extends IgxMaskDirective implements OnCh
       this.updateMask();
     }
 
+    // TODO: remove when formatter func has been deleted
+    if (this.displayValuePipe) {
```

Comment:
We need it because we don't want to call the mask's blur handler if a `displayValuePipe` is set. Because in the case with the date picker, the pipe will call the `formatter` function which is supposed to manipulate dates, while the argument for the mask's blur handler is a `string`. This will be removed as soon as the `formatter` is no longer present in the date picker.

Reply:
Still `displayValuePipe` functionality is inherited, apparently not very well though. Should be improved in the future

Reply:
Still displayValuePipe functionality is inherited, apparently not very well though. Should be improved in the future